squad/core/migrations/0127_metric_thresholds_migrate_data.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.6 on 2020-02-26 14:42
+
+from django.db import migrations
+
+
+def populate_env_in_thresholds_and_migrate_olddata(apps, schema_editor):
+    Threshold = apps.get_model('core', 'MetricThreshold')
+    Environment = apps.get_model('core', 'Environment')
+    existing_thresholds = Threshold.objects.all().iterator()
+    for threshold in existing_thresholds:
+        envs = Environment.objects.filter(project_id=threshold.project_id)
+        for env in envs:
```

Comment:
Another good question man :) which i asked myself. Well seems like the best approach was to copy old data regardless of which env it logically belongs to. And then users can delete the non-relevant ones manually later which is easy. Just in case of any squad prod instances used by members for example, where this will just retain their defined thresholds. Blanket approach but makes sense i guess.

Reply:
@stevanradakovic any other ideas how to convert existing thresholds to the new model?

Reply:
@stevanradakovic I realized i was answering about migrations and not thresholds for each env. sorry for wrong read.
Talking with LKFT guys, they want the scope of a threshold to be tied to the smallest unit possible (eg. env) for it to make sense in their calculations. Using projects is too wide for it to make sense, and hence per env made sense. If you got other thoughts, please let me know.

Reply:
@mwasilew not really I was just thinking out loud, to give you guys some food for thought.